pkg/cpu/cpu.go

```diff
@@ -0,0 +1,78 @@
+/*
+ * Minio Cloud Storage, (C) 2016, 2017, 2018 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cpu
+
+import (
+	"fmt"
+	"sync"
+	"time"
+)
+
+// Performance holds CPU utilization % measured in three intervals of 200ms each
+type Performance struct {
+	Avg   string `json:"avg"`
+	Max   string `json:"max"`
+	Min   string `json:"min"`
+	Error string `json:"error,omitempty"`
+}
+
+type counter struct{}
+
+// GetPerformance returns the CPU utilization % of the current process
+func GetPerformance() Performance {
+	vals := make(chan time.Duration, 3)
+	wg := sync.WaitGroup{}
+	for i := 0; i < 3; i++ {
+		cpuCounter, err := newCounter()
+		if err != nil {
+			return Performance{
+				Error: err.Error(),
+			}
+		}
+		go func() {
+			wg.Add(1)
+			start := cpuCounter.now()
+			time.Sleep(200 * time.Millisecond)
```

Comment:
All constants should be global constants with some meaning/comment on why this value, if it's arbitrary just say so that it can be any other value as well.